src/components/Firebase/firebase.js

```diff
@@ -17,13 +18,17 @@ class Firebase {
 
     /* Helper */
 
-    this.serverValue = app.database.ServerValue;
+    this.fieldValue = app.firestore.FieldValue;
     this.emailAuthProvider = app.auth.EmailAuthProvider;
 
     /* Firebase APIs */
 
     this.auth = app.auth();
-    this.db = app.database();
+
+    /* FireStore */
```

Comment:
Let's remove this comment and have it as before aggregated under the /* Firebase APIs */  comment.